src/Microsoft.DocAsCode.AzureMarkdownRewriters/AzureMarked.cs

```diff
@@ -26,17 +26,17 @@ public class AzureMarked
                 context = engine.Context.CreateContext(engine.Context.Variables.SetItem("path", path));
             }
 
-            if (azureMarkdownFileInfoMapping != null)
+            if (azureMarkdownFileInfoMapping?.Count != 0)
```

Comment:
Just not set the video info in the context and will skip the video transform.

---
In reply to: [73485155](https://github.com/dotnet/docfx/pull/564#discussion_r73485155) [](ancestors = 73485155)